exporter/exporterhelper/xexporterhelper/profiles_test.go

```diff
@@ -57,6 +57,20 @@ func TestProfilesRequest(t *testing.T) {
 	)
 }
 
+func TestProfilesEncoding_UnmarshalError(t *testing.T) {
```

Comment:
Same for https://app.codecov.io/gh/open-telemetry/opentelemetry-collector/pull/12802/indirect-changes

Reply:
Removed [moving and renaming featuregate and added otelcol_receiver_requests u…](https://github.com/open-telemetry/opentelemetry-collector/pull/12802/commits/89d7bc412c39d753f5b206a8b103931c66cba53e)